data/mods/DinoMod/monsters/laser_dinos_zinos.json

```diff
@@ -102,7 +102,7 @@
         "no_ammo_sound": "*zZzzz*-TARGET ACQUIRED.  *beeeeeeeeep*.  OUT OF AMMO.-*zZzZ*"
       }
     ],
-    "description": "A shiny finned dinosaur corpse covered with bloody armor and crackling bionics.  It shambles quickly as it hunts.  Is...  is that a laser rifle on its head?",
+    "description": "A shiny finned dinosaur corpse covered with bloody armor and crackling bionics.  It shambles quickly as it hunts.  Is that a laser rifle on its head?",
```

Comment:
The warning was about using ellipsis (`…`) instead of three dots (`...`), which was why adding a space didn't solve it.

Reply:
didnt know there was an ellipsis character. thanks, but i think the description could do without the three dots / ellipsis, as i think it is a bit cheesy in my desc.